hello.py

```
@@ -1 +1 @@
-print("hello coderabbit")
+print_error("hello coderabbit")
```

Ruff (0.8.2)

Undefined name `print_error`

(F821)